enterprise/models.py

```diff
@@ -2461,6 +2461,103 @@ class LicensedEnterpriseCourseEnrollment(EnterpriseFulfillmentSource):
     )
 
 
+class DefaultEnterpriseEnrollmentIntention(TimeStampedModel, SoftDeletableModel):
+    """
+    Specific to an enterprise customer, this model defines a course or course run
+    that should be auto-enrolled for any enterprise customer user linked to the customer.
+
+    .. no_pii:
+    """
+    DEFAULT_ENROLLMENT_CONTENT_TYPE_CHOICES = [
+        ('course', 'Course'),
+        ('course_run', 'Course Run'),
+    ]
+    uuid = models.UUIDField(
+        primary_key=True,
+        default=uuid4,
+        editable=False,
+    )
+    enterprise_customer = models.ForeignKey(
+        EnterpriseCustomer,
+        blank=False,
+        null=False,
+        related_name="default_course_enrollments",
+        on_delete=models.deletion.CASCADE,
+        help_text=_(
+            "The customer for which this default enrollment will be realized.",
+        )
+    )
+    content_type = models.CharField(
+        max_length=127,
```

Comment:
nit/curious: Why `127` here?

Reply:
(2^7 - 1). i.e the same reason we use `max_length` of 255 instead of 256. I'm not sure how much this matters, if it matters at all. There used to be some thing about using one fewer chars than a power of two to make page fetches more efficient (I think).

Reply:
Interesting, TIL haha.